cmake/CMakeLists.txt

```diff
@@ -594,7 +577,7 @@ if(PKG_ATC)
   if(BUILD_MPI)
     target_link_libraries(atc PRIVATE MPI::MPI_CXX)
   else()
-    target_link_libraries(atc PRIVATE mpi_stubs)
+    target_include_directories(atc PRIVATE ${LAMMPS_SOURCE_DIR}/STUBS)
```

Comment:
I don't like the fact the we explicitly have to hard-code this directory everywhere.

I think slightly more CMake-ish way of doing this would be to create add target:
```cmake
add_library(mpi_flags INTERFACE)
target_include_directories(mpi_flags INTERFACE ${LAMMPS_SOURCE_DIR}/STUBS)
```
and then use
```
target_link_libraries(atc PRIVATE mpi_flags)
```

Reply:
I don't like this indirect "CMake way" of specifying include paths by "faking" a library. Just adding an include path is more obvious to non-experts. However, if that "fake" library is called under the same target as the real MPI library, then this would allow to treat both cases as "need to add MPI support here" and reduce some redundant calls. Made some changes along that way and that seems to be working as expected.

Reply:
Good point! mpi_flags makes it more clear it depends on mpi and not something else in stubs.